components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/util/OAuth2UtilTest.java

```diff
@@ -1799,4 +1969,315 @@ public void testValidateRequestTenantDomain(boolean isTenantQualifiedURLsEnabled
 
         assertEquals(isInvalidClientExceptionThrown, isExceptionExpected);
     }
+
+    @DataProvider(name = "createResponseType")
+    public Object[][] createResponseType() {
+
+        return new Object[][]{
+                {"", false},
+                {null, false},
+                {OAuthConstants.CODE_TOKEN, true},
+                {OAuthConstants.CODE_IDTOKEN, true},
+                {OAuthConstants.CODE_IDTOKEN_TOKEN, true}
+        };
+    }
+
+    @Test(dataProvider = "createResponseType")
+    public void testisHybridResponseType(String responseType, boolean expected) {
```

Comment:
Fix improper camel casing in method name  `testisHybr...` -> `testIsHybr...`.

Reply:
Addressed with commit [9e575a3](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1691/commits/9e575a323c0604e9985d430a5b3fee4942d86cb1)